### PR TITLE
Added styling support for markdown images

### PR DESCRIPTION
**Pull request by eswarasai**
### Description
- [x] Added support for inline props for the images
   - [x] Size: full-width or specific size
   - [x] Alignment: center full screen or inline with text left or right

Default is `width: 100%` and center aligned.
For width specification, add a url parameter `?width=400px`.
For content and image alignment, add a url parameter `?float=left`.

Sample URL with both specific width and content alignment `?width=400px&float=right`.

- [x] Added support for static images

Fixes: #141

### CODE REVIEW

**Comment by charleenfei** [comment]
@eswarasai, thanks for this! one change that i would request (which I saw you have already marked as a TODO) is that this issue requests support for static image assets. Can you update the PR to include this?

**Comment by eswarasai** [comment]
@charleenfei - I've added the support to render images from `src/images` folder within the Markdown using this plugin -- https://www.gatsbyjs.org/packages/gatsby-remark-images/

But unfortunately, it'll be pretty difficult to support the custom styling like size and alignment because of the external plugin. Let me know what would you suggest the path going forward. Thanks!

**Comment by charleenfei** [comment]
@eswarasai, it would be great if you could just take the current images that we are fetching dynamically and add them to the `src/images` directory just with inline styling so we are just working with static assets, then I would consider the ticket done. We can update it from there before merging to production. 

Thank you!

**Comment by eswarasai** [comment]
@charleenfei - Just pushed an example commit which demonstrates the usage of CSS styling like suggested. Also, this is a pretty good source to find such hacks for better styling of images with re-usable styles etc. -- https://www.xaprb.com/blog/how-to-style-images-with-markdown/

Quick demo: https://deploy-preview-144--staging-centrifuge-documentation.netlify.com/tinlake/ui/overview/

Let me know what you guys have decided so that I can wrap up this one. Thanks 🙂

**Comment by charleenfei** [comment]
@eswarasai, cool, please go ahead with this solution then for the rest of the images.

**Comment by eswarasai** [comment]
@charleenfei - Just pushed the changes. Let me know if there's anything else needed here. Thanks!

File: docs/tinlake/overview/introduction.md [hunk 1]
```diff
@@ -21,4 +21,4 @@ Tinlake interacts with borrowers and lenders. Generally, borrowers have individu
 Generally, an administrator determines what assets borrowers are allowed to bring in as collateral and how much they are allowed to borrow against the assets. This administrator could be a single centralized entity, a set of smart contracts automatically qualifying and pricing an asset or a decentralised autonomous organisation (DAO) that manages the Tinlake deployment.
 
 ## System Overview
-![tinlake overview](https://centrifuge.io/static/tinlake-work-desktop-23ec0d1949774739533ca02062f8c3cc.svg)
```
Comment (charleenfei) [inline docs/tinlake/overview/introduction.md]: this link looks broken in the deploy preview

Comment (eswarasai) [reply docs/tinlake/overview/introduction.md]: @charleenfei - This should be fixed now. Thanks for pointing it out 🙂

**Pull request merged**